scrapers/Mofos.yml

```diff
@@ -1,49 +1,17 @@
 name: mofos
 performerByURL:
   - action: scrapeXPath
-    url: 
+    url:
       - mofos.com/model
     scraper: performerScraper
-sceneByURL:
-  - action: scrapeXPath
-    url: 
-      - mofos.com/scene
-    scraper: sceneScraper
 xPathScrapers:
   performerScraper:
     performer:
       Name: //h1
```

Comment:
Is there a use for a performer scraper only fetching the name ?
Since the main, scene functionality is in the RK mega scraper this one Mofos.yml should be removed alltogether.

Reply:
Honestly not sure, I wasn't sure if anyone was using this functionality. It looked like it was still being worked on with all the comments so I wasn't sure if I should leave it or remove it. Think I should remove it entirely then?

Reply:
Yeah, remove it. If anyone is interested they can make a new one only for the performers.

Reply:
Done!